packages/generator-single-spa/src/root-config/templates/src/index.ejs

```diff
@@ -17,6 +17,9 @@
   -->
   <meta http-equiv="Content-Security-Policy" content="default-src 'self' https: localhost:*; script-src 'unsafe-inline' https: localhost:*; connect-src https: localhost:* ws://localhost:*; style-src 'unsafe-inline' https:; object-src 'none';">
   <meta name="importmap-type" content="systemjs-importmap" />
+  <!-- If you wish to turn off import-map-overrides for specific environments (prod), uncomment the line below -->
+  <!-- More info at https://github.com/joeldenning/import-map-overrides/blob/master/docs/configuration.md#domain-list -->
```

Comment:
The link here won't work until https://github.com/joeldenning/import-map-overrides/pull/36 is merged. In the meantime, see https://github.com/joeldenning/import-map-overrides/blob/9aa8079d051fef16e3027e0917837f3a86b18e79/docs/configuration.md#domain-list